code/object/collideshipship.cpp

```diff
@@ -751,17 +792,23 @@ void calculate_ship_ship_collision_physics(collision_info_struct *ship_ship_hit_
 	// We will try not to worry about the left over time in the frame
 	// heavy's position unchanged by collision
 	// light's position is heavy's position plus relative position from heavy
-	vm_vec_add(&lighter->pos, &heavy->pos, &ship_ship_hit_info->light_collision_cm_pos);
+	if (should_collide){
```

Comment:
Why are there 3 separate if statements with the same condition? Looks like the first two could be combined at least.

Reply:
Combined. Yeah, there's absolutely no reason to check twice for the first two.